app/javascript/src/components/Dashboard/Notes/NotesList.jsx

```diff
@@ -40,7 +40,7 @@ const NotesList = ({ note, setSelectedNoteIds, setShowDeleteAlert }) => {
             position="bottom"
           >
             <Typography style="body3">
-              {`Created ${calculateCreatedAgo(note.created_at)} `}
```

Comment:
Why was this change made?

Reply:
I have already corrected this. I'll check again why did this happen.